### DISCUSSION TITLE
No support for "Shipmondo for WooCommerce" extension.

### DISCUSSION THREAD

**Discussion by mhmoeller** [woocommerce/woocommerce#45068] [WooCommerce, Blocks, and Site Editing]
## What do you think is missing from the Cart & Checkout blocks?

The 'shipmondo for woocommerce' plugin works with normal woocommerce checkout page. But doesn't work with the custom one from the blocks.
Usually shipmondo outputs a modal or dropdown pickup map to let the customer choose where to pick up their package.

**Comment by nerrad** [comment]
Hi there @mhmoeller  👋 

Thanks for taking the time to make us aware that Shipmondo for WooCommerce extension isn't working with the Checkout and Cart blocks. These blocks are still in early access (via the feature plugin) and it's expected that some existing Woo Extensions that integrate with the current checkout flow in WooCommerce core will not work with the blocks. 

We're currently working on adding extensibility for the blocks so extensions have a way to make sure they integrate with the new flows being developed. Once that is done we'll have docs prepared and will be reaching out to Woo Extension developers to help them with getting their extensions integrated.

There is already [some documentation available here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/trunk/docs/extensibility) that you could share with the developers of that extension now if you want to get this on their radar!

Until then you can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/).

We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don’t note or review the feedback. Closed issues can also still be commented on.

Thanks again for leaving the feedback!
